src/textOnlyPackages/src/microsoft.dotnet.web.itemtemplates/5.0.0/.nupkg.metadata

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
I don't think this file should be included.

Reply:
Huh... not sure how I missed this one. It's also missing the csproj.

Reply:
VS Code is acting a little oddly with detecting file system changes right now. The manual refresh button wasn't quite working either--had to do a window reload. Should be good now though.